### PR TITLE
Explanatory tile for how to integrate with Discord using webhooks

### PR DESCRIPTION
**Pull request by maxandaker**
### What does this PR do?

Adds and explanatory tile for how to integrate with Discord using webhooks.

### Motivation

Customers inquiring about a Discord integration are not always aware that they can set this up using the webhook option.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [N/A] Feature or bugfix has tests
- [x] Git history is clean
- [N/A] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [N/A] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

As discussed with @moreauowen I'm opening this PR to preview this tile in Staging.
https://dd.slack.com/archives/C054L3BC0MT/p1710430113821319?thread_ts=1710430049.094689&cid=C054L3BC0MT

### CODE REVIEW

File: discord/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+4. You can customize your payload or copy-paste the template below. See the [webhook documentation](https://docs.datadoghq.com/integrations/webhooks/#usage) for more details on available variables.
+
+```json
+{
+   "embeds":[
+      {
+         "title":"$EVENT_TITLE",
+         "description":"$EVENT_MSG",
+         "url":"$LINK",
+         "color":"3407966",
```
Comment (aTegart) [inline discord/README.md:24]: What color is this? Do we want to change the color between red and green depending on whether the monitor is alerting or recovering? (Monitor messages supporting that sort of conditional switching, but I'm not sure if the webhook integration does)

Comment (maxandaker) [reply discord/README.md:24]: As of now, this is just the official Datadog purple from our brand guidelines. I don't know how to make this conditional without creating to variants and then describe a condition at the monitor-level. I think that makes it unnecessarily complicated for the sake of this exercise though.

File: discord/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            "url":"$SNAPSHOT"
+         }
+      }
+   ]
+}
+```
+
+## Troubleshooting
+
+Need help? Contact [Datadog support](https://docs.datadoghq.com/help/)
```
Comment (maycmlee) [inline discord/README.md:35]:
```suggestion
Need help? Contact [Datadog support][1]

[1]: https://docs.datadoghq.com/help/
```

**Review by maycmlee — COMMENTED** [review]
Left some suggestions!

**Review by maycmlee — APPROVED** [review]

**Review by anawishnoff — APPROVED** [review]

**Pull request merged**